common/data/base/houseBase.ts

```diff
@@ -0,0 +1,17 @@
+﻿/* eslint-disable */
+import {getDataJsonByUrl, getObjectsTotalCount, StrapiBaseUrl} from "../dataHelpers";
+import {CanGetById} from "./canGetById";
+
+
+export class HouseBase extends CanGetById{
```

Comment:
Let's simplify classes to objects for better readability

```
export const houseBase = {
    getObject: () => {
       return ...
    },
    getObjectsPolygonsByRange: () => {
       return ...
    }
}
``

Reply:
maybe if house is a class, then the base should also be one?